src/bosh_azure_cpi/lib/cloud/azure/models/config.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+module Bosh::AzureCloud
+  class AzureStackConfig
+    attr_reader :domain, :authentication, :resource, :endpoint_prefix
+    attr_writer :authentication
+    def initialize(azure_stack_config_hash)
+      @domain = azure_stack_config_hash['domain']
+      @authentication = azure_stack_config_hash['authentication']
+      @resource = azure_stack_config_hash['resource']
+      @endpoint_prefix = azure_stack_config_hash['endpoint_prefix']
+    end
+  end
+
+  class AzureConfig
+    include Helpers
+    attr_reader :request_id, :use_managed_disks, :enable_telemetry
```

Comment:
request_id has nothing to do with Azure, it is weird to put it here.

Reply:
Also suggest to remove request_id from AzureConfig.